cmd/flux/main.go

```diff
@@ -99,29 +99,32 @@ Command line utility for assembling Kubernetes CD pipelines the GitOps way.`,
 var logger = stderrLogger{stderr: os.Stderr}
 
 type rootFlags struct {
-	kubeconfig   string
-	kubecontext  string
-	namespace    string
 	timeout      time.Duration
 	verbose      bool
 	pollInterval time.Duration
 	defaults     install.Options
 }
 
 var rootArgs = NewRootFlags()
+var kubeconfigArgs = genericclioptions.NewConfigFlags(false)
 
 func init() {
-	rootCmd.PersistentFlags().StringVarP(&rootArgs.namespace, "namespace", "n", rootArgs.defaults.Namespace,
```

Comment:
We need to preserver the default value from `rootArgs.defaults.Namespace` otherwise we'll break everyone's clusters. Flux default namespace is `flux-system` not `default`.

Reply:
Yeah, I've just fixed that with the force push :smile: